File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L2/08/Level2.py
import sys

def balik(angka):
    while angka != 0:
        sisa = angka % 10
        print(sisa, end='')
        angka = angka // 10
    print()

if __name__ == '__main__':
    print("Enter an integer: ", end='')
    angka = int(input())
    balik(angka)

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L2/08/Level2.kt
package IR

/**
 * @author FD8DBE9073AFCC0504CD6901E1091CAD
 */
object Level2 {
    @JvmStatic
    fun main(args: Array<String>) {
        println("Enter an integer: ")
        val `in` = java.util.Scanner(System.`in`)
        val angka = `in`.nextInt()
        balik(angka)
    }

    //fungsi membalik
    private fun balik(angka: Int) {
        var temp = angka
        while (temp != 0) {
            val sisa = temp % 10
            print(sisa)
            temp /= 10
        }
        println()
    }
}

